<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrqTrigChan</class>
 <widget class="QWidget" name="MrqTrigChan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLevel">
      <attribute name="title">
       <string>Level</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkLevel">
         <property name="text">
          <string>ON/OFF</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Level Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbLevelMode">
         <item>
          <property name="text">
           <string>RESERVE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LOW</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RISE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HIGH</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Level Response</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmbLevelResponse">
         <item>
          <property name="text">
           <string>NONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALARM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STOP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALARM + STOP</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="spinLevelPeriod">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPattern">
      <attribute name="title">
       <string>Pattern</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ON/OFF</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
